src/clr-angular/data/tree-view/tree-features.service.ts

```diff
@@ -15,6 +16,7 @@ export class TreeFeaturesService<T> {
     template: TemplateRef<ClrRecursiveForOfContext<T>>;
     root: RecursiveTreeNodeModel<T>[];
   };
+  childrenFetched: Subject<void> = new Subject();
```

Comment:
It's not because models need to emit their children fetched events from it. As I didn't want very node to subscribe to each of it's children, I decided to extract it to a common service where one observable can serve all updates. As the final goal is to trigger change detection it doesn't really matter which children have been fetched.

Reply:
I think its ok.